src/server_manager/ui_components/outline-server-settings.html

```diff
@@ -93,19 +87,48 @@
       paper-input {
         /* Removes extra padding added by children of paper-input */
         margin-top: -8px;
+        --paper-input-container-label-focus: {
+          color: var(--primary-green);
+        };
+        --paper-input-container-underline-focus: {
+          border-color: var(--primary-green);
+        };
+        --paper-input-container-label: {
+          font-size: 14px;
+          line-height: 22px;
+        };
+      }
+      paper-input[disabled] {
+        --paper-input-container-underline: {
+          display: none;
+        };
+        --paper-input-container-underline-focus: {
+          display: none;
+        };
+        --paper-input-container-underline-disabled: {
+          display: none;
+        };
+        --paper-input-container-disabled: {
+          opacity: 1;
+        }
       }
     </style>
     <div class='fullScreenOverlay' hidden$='[[!visible]]'>
       <app-toolbar>
-        <h2>Server Settings</h2>
+        <paper-icon-button icon="arrow-back" on-tap="save"></paper-icon-button>
+        <h2>Server Details</h2>
       </app-toolbar>
       <div class='content'>
-        <!-- Rename controls -->
         <div class='setting'>
-          <iron-icon icon="create"></iron-icon>
+          <iron-icon icon="outline-iconset:outline"></iron-icon>
           <div>
-            <paper-input value='{{name}}' label='name' always-float-label maxlength='100'></paper-input>
-            <p>Set a new name for your server. Note that this will not be reflected on the devices of the users that you invited to connect to it.</p>
+            <h3>Information</h3>
+            <paper-input value='{{name}}' label='Name' always-float-label maxlength='100'></paper-input>
+            <p class='detail'>Set a new name for your server. Note that this will not be reflected on the devices of the users that you invited to connect to it.</p>
+            <paper-input disabled value='[[serverHostname]]' label='Hostname' always-float-label maxlength='100'></paper-input>
```

Comment:
For good measure, you should to hide fields that may not exist. Just checking if the value is valid is probably ok.

Think if it as a reusable widget that others may use in their own application where our invariants may not be valid.

Reply:
Done in 117fcac. FYI, I had to initialize the properties to `null` because the Polymer hidden attribute does not work with `undefined` values.